src/tests/api/HappyPath.test.ts

```diff
@@ -15,14 +15,86 @@ describe("Happy path tests", () => {
 			const sessionResponse = await startStubServiceAndReturnSessionId(journeyType);
 			const sessionId = sessionResponse.data.session_id;
 
-			const savedJourney = getKeyFromSession(sessionId, constants.DEV_CIC_SESSION_TABLE_NAME, "journey");
-			await expect(savedJourney).resolves.toBe(journeyType);
+			await getSessionAndVerifyKey(sessionId, constants.DEV_CIC_SESSION_TABLE_NAME, "journey", journeyType);
+			await getSessionAndVerifyKey(sessionId, constants.DEV_CIC_SESSION_TABLE_NAME, "authSessionState", "CIC_SESSION_CREATED");
 
 			const allTxmaEventBodies = await getTxmaEventsFromTestHarness(sessionId, 1);
 			validateTxMAEventData({ eventName: "CIC_CRI_START", schemaName }, allTxmaEventBodies);
 		});
 	});
 
+	describe("/claimedIdentity endpoint", () => {
+		it.each([
+			{ journeyType: "FACE_TO_FACE", schemaName: "CIC_CRI_START_SCHEMA" },
+			{ journeyType: "NO_PHOTO_ID", schemaName: "CIC_CRI_START_BANK_ACCOUNT_SCHEMA" },
+		])("Successful Request Tests - $journeyType", async ({ journeyType, schemaName }: { journeyType: string; schemaName: string }) => {
+			const sessionResponse = await startStubServiceAndReturnSessionId(journeyType);
```

Comment:
Again, maybe a little out of scope of the ticket so feel free to shut it down BUT the function is called `startStubServiceAndReturnSessionId` but it doesn't actually return a session ID? It's type is that it returns `any` and on the line below we have to access `sessionResponse.data.session_id`. 

What do you think about updating the function so that it actually return the session ID and the response status (looks like that is being also used)?

Reply:
Yeah that makes sense, I've refactored teh method so that it now returning the sessionId and updated the test logic to reflect this. 

Test Results (after run following changes in previous comment):
![image](https://github.com/govuk-one-login/ipv-cri-cic-api/assets/110032361/41d4cfe9-7367-4c77-9d7a-c0a860a6569f)